documentation/contributing/releases/index.markdown

```diff
@@ -0,0 +1,111 @@
+---
+author: tylerjw
+layout: page
+slug: releases
+title: Releases
+---
+
+# Release Process
+
+MoveIt has a 6 week cadence for releases.  This pace is a balance between releasing new improvements quickly and preserving the stability of releases.  We are choosing this schedule to reduce the amount of work per release, reduce the time it takes to get an improvement released and make it easier to schedule for the Picknik team.  These releases will include the main MoveIt repo along with any dependencies we control if they have changed (e.g. moveit_msgs).
+
+## Upcoming releases
+
+The release schedule is avalible via [this Google Calendar](https://calendar.google.com/calendar/embed?src=c_bpueq5jcajuaejr2di0hdr4qgg%40group.calendar.google.com&ctz=America%2FDenver).
+
+### Typical Release Schedule
+
+* Friday: Begin preparing release
+* Following Friday: Bloom
+* Tuesday After Bloom: Public Announcement
+
+## Permissions
+
+The maintainer handling the release must have write access to both devel repos (e.g. [github.com/ros-planning/moveit](https://github.com/ros-planning/moveit)) as well as release repos (e.g. [github.com/ros-gbp/moveit-release](https://github.com/ros-gbp/moveit-release)). The release repo access is managed via the Github Team [moveit-maintainers](https://github.com/orgs/ros-gbp/teams/moveit-maintainers/members).
+
+## Steps
+
+1. Create an issue to track the status of the upcomming release.  This issue should include a list of the changes since the last release so maintainers can determine which ones should be backported into the release branch.  Do this for each repo that will be released with references to the issue in the main repo.  To generate the list of changes use the `review-for-backport` branch. The `reviewed-for-backport` branch exists as a marker in the commit history for `master`, marking the last commit that was considered for backporting. Run `git cherry -v <release-branch> master reviewed-for-backport | grep '^+'` to generate a list of all commits between `reviewed-for-backport` and the current `master` branch that have not been merged in `release-branch` (without modifications). Backports should be pull-requests and reviewed separately, though possibly bundled.
```

Comment:
```suggestion
1. Create an issue to track the status of the upcoming release.  This issue should include a list of the changes since the last release so maintainers can determine which ones should be backported into the release branch.  Do this for each repo that will be released with references to the issue in the main repo.  To generate the list of changes use the `review-for-backport` branch. The `reviewed-for-backport` branch exists as a marker in the commit history for `master`, marking the last commit that was considered for backporting. Run `git cherry -v <release-branch> master reviewed-for-backport | grep '^+'` to generate a list of all commits between `reviewed-for-backport` and the current `master` branch that have not been merged in `release-branch` (without modifications). Backports should be pull-requests and reviewed separately, though possibly bundled.
```